pkg/utils/validation_test.go

```diff
@@ -0,0 +1,23 @@
+package utils
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+)
+
+var _ = Describe("Check OpenShift Existance Validation", func() {
+	testEnv := &envtest.Environment{}
```

Comment:
Any reason to not put this into BeforeSuite, just like we always do?
Maybe also configure the logger?